mmv1/third_party/terraform/services/backupdr/data_source_backup_dr_backup_plan_test.go

```diff
@@ -78,3 +99,57 @@ data "google_backup_dr_backup_plan" "fetch-bp" {
   }
 `, context)
 }
+
+func testAccDataSourceGoogleBackupDRBackupPlan_csql(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_backup_dr_backup_vault" "my-backup-vault-csql" {
+    location ="us-central1"
+    backup_vault_id    = "bv-%{random_suffix}"
```

Comment:
```suggestion
    backup_vault_id    = "tf-test-bv-%{random_suffix}"
```
We'll need `tf-test` prefix for resource identifier so that our sweeper will clean up the dangling resources if it leaks

Reply:
Done